packages/cli/src/dev-server/http-server.ts

```diff
@@ -74,7 +74,7 @@ export class DevServer {
         <link href="${this.options.dashboardUrl}/favicon.svg" rel="icon" />
         <title>Novu Studio</title>
       </head>
-      <body style="padding: 0; margin: 0;">
+      <body style="padding: 0; margin: 0; overflow: hidden;">
```

Comment:
Disables all scrolling on the root element of Local Studio, so that scrolling does not move the page up and down when there is nothing scrollable.